ush/fv3gfs_driver_grid.sh

```diff
@@ -468,6 +486,20 @@ elif [ $gtype = regional_gfdl ] || [ $gtype = regional_esg ]; then
 
 fi
 
+
+
+
+#------------------------------------------------------------------------------------
+#------------------------------------------------------------------------------------
+#Redo orog and filter.
+#------------------------------------------------------------------------------------
```

Comment:
Why is the orography re-created?

Reply:
As per notes , the lake mask is derived first using the step 1 and when the ocean mask is merged, the lake points may change and hence is re run.  the lake merge routine needs the oro files.

Reply:
Ok. I would add a few comments to the script for clarity. Also, I would document this multi-step process in the issue (#825).

Reply:
I have added some explanatory texts to both scripts and the issue